README.md

```diff
@@ -165,15 +165,15 @@ git checkout master
 8. Add the next minor release to the ``CHANGES`` file:
 
 ```
-## 1.10.0 (unreleased)
+## 1.11.0 (unreleased)
```

Comment:
On the section above this one the referenced version has been updated to `2.x.x` but here it's still `1.x.x`